README.md

```diff
@@ -1,6 +1,36 @@
-Current repo contain the set of tests, adopted for execution in [saveourtool](https://saveourtool.com/)
+Current repo contain the set of tests, adopted for execution in [saveourtool](https://saveourtool.com/) platform.
 
-For local execution you need to run following commands **in the root of project**
+### Local run
 
-    ./setup.sh
-    ./save-0.3.9-linuxX64.kexe . --log all --report-type json --result-output file
\ No newline at end of file
+1) Extract the files from `executable` directory to the root of project.

```

Comment:
maybe refactor it and move all tests to subfolder?

Reply:
+++++++

Reply:
We are right now arguing about this

Reply:
good luck then!)